lib/Client.js

```diff
@@ -601,8 +602,10 @@ class Client extends EventEmitter {
                 };
             } else if(content.content !== undefined && typeof content.content !== "string") {
                 content.content = "" + content.content;
-            } else if(content.content === undefined && !content.embed && !file) {
-                return Promise.reject(new Error("No content, file, or embed"));
+            } else if(content.content === undefined && !content.embed && !content.embeds && !file) {
```

Comment:
I'd rather have the deprecated prop reassigned into an `embeds` array instead of still sending it through to Discord